### ISSUE TITLE
Cannot save in Windows

### ISSUE THREAD

**Issue by mflorence99** [mflorence99/lintel#25]
@jerone reports:

```
[2020-11-06 19:07:17.174] [renderer1] [error] ENOENT: no such file or directory, open 'c:Usersjeronesource
eposFrontendClientApp.eslintrc.json': Error: ENOENT: no such file or directory, open 'c:Usersjeronesource
eposFrontendClientApp.eslintrc.json'
    at Object.openSync (fs.js:440:3)
    at Object.func [as openSync] (electron/js2c/asar.js:140:31)
    at Object.writeFileSync (fs.js:1265:35)
    at Timeout._onTimeout (c:\Users\jerone\.vscode\extensions\mflo999.lintel-1.3.0\ext\message-handler.ts:21:10)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
[2020-11-06 19:07:48.721] [renderer1] [error] cannot open c:Usersjeronesource%0DeposFrontendClientApp.eslintrc.json. Detail: Unable to resolve resource c:Usersjeronesource%0DeposFrontendClientApp.eslintrc.json: Error: cannot open c:Usersjeronesource%0DeposFrontendClientApp.eslintrc.json. Detail: Unable to resolve resource c:Usersjeronesource%0DeposFrontendClientApp.eslintrc.json
    at file:///C:/Users/jerone/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:6688:388
```

**Comment by jerone** [comment]
In the UI I see exactly the same as in #14; one long name of the full path, but without any slashes.

**Comment by mflorence99** [comment]
Sorry for delay. I'm beginning to understand that I never really fixed #14 -- I didn't properly follow up when I misunderstood its cause. My issue is that I don't have access to Windows myself. Instead, I have a unit test case `assets/eslint-files/windows.ts` that *simulates* files coming from Windows. 

Clearly that's not sufficient. On reflection, the \ path separator can't be interpreted as an escape, because if it were we wouldn't see `c:Users...` for `c:\Users\...`.

**Comment by mflorence99** [comment]
Just published v1.3.1 which should fix #24 and #25. #26 is a bit tricky and I have to think about it. I just wanted to get @jerone started in light of the fact I don't really have a Windows system.
